shared/src/main/scala/mouse/iterableonce.scala

```diff
@@ -0,0 +1,13 @@
+package mouse
+
+import compat.scalaVersionSpecific._
+
+trait IterableOnceSyntax {
+  implicit final def iterableOnceStringSyntaxMouse(i: IterableOnce[String]): IterableOnceStringOps =
+    new IterableOnceStringOps(i)
+}
+
+final class IterableOnceStringOps(private val i: IterableOnce[String]) extends AnyVal {
```

Comment:
I think that is quite specialized for having this in `mouse`. Roughly speaking, seems this is an implementation detail of `FoldableSyntax#unlines_`.

Reply:
@danicheg I think that there exist both `Foldable` instance that does not extend from `IterableOnce` and `IterableOnce` subtypes that don't have `Foldable` instance, so it's still good to expose both.